packages/trpc/server/routers/viewer/organizations/createTeams.handler.ts

```diff
@@ -238,6 +240,34 @@ async function moveTeam({
     teamSlug: newSlug,
     orgSlug: org.slug || (orgMetadata?.requestedSlug ?? null),
   });
+
+  // Cancel existing stripe subscriptions once the team is migrated
+  const subscriptionId = getSubscriptionId(team.metadata);
+  if (subscriptionId) {
+    await tryToCancelSubscription(subscriptionId);
+  }
+}
+
+function tryToCancelSubscription(subscriptionId: string) {
```

Comment:
```suggestion
async function tryToCancelSubscription(subscriptionId: string) {
```

Reply:
OH DERP Idk why my ide didnt bitch about this

Reply:
We certainly have some config missing in eslint that could have caught this.